InventoryWatch/MenuBarView.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  MenuBarView.swift
+//  InventoryWatch
+//
+//  Created by Ramik Sadana on 09/28/22.
+//
+
+import SwiftUI
+
+struct MenuBarView: View {
+    @EnvironmentObject var model: Model
+    
+    @AppStorage("lastUpdateDate") private var lastUpdateDate: String = ""
+    
+    var body: some View {
+        ScrollView(.vertical) {
+            VStack(alignment: .leading, spacing: 5) {
+                    if let error = model.errorState {
+                        Text(error.errorMessage)
+                            .font(.subheadline)
+                            .italic()
+                            .padding(.bottom)
+                    }
+                
+                VStack(alignment: .leading, spacing: 5) {
+                    
+                    let storeFont = Font.headline.weight(.semibold)
+                    let cityFont = Font.subheadline.weight(.semibold)
+                    let productFont = Font.body.weight(.regular)
```

Comment:
not sure if this needs to be a part of your PR, but I'd love to have more code shared between `MenuBarView` and `ContentView`, since they're doing extremely similar things.

Reply:
I first tried the original style, and the bold/semibold format just didn't look good for the menu bar. I agree though that eventually it should just be one unified style encapsulated in, e.g., a `DesignSystem` file. But you should probably be the one bringing all that together.